src/Gui/PythonWrapper.cpp

```diff
@@ -470,6 +470,9 @@ qttype* qt_getCppType(PyObject* pyobj)
     // https://github.com/PySide/Shiboken/blob/master/shibokenmodule/typesystem_shiboken.xml
     Py::Module mainmod(importShiboken(), true);
     Py::Callable func = mainmod.getDict().getItem("getCppPointer");
+    if (func.isNull()) {
+        throw Py::RuntimeError("Failed to get C++ pointer");

```

Comment:
I put a note to myself here: I was temped to change this into `return nullptr;` to be consistent with Shiboken C++ version, but this is indeed runtime error. Perhaps these two functions (`getCppPointer` and `wrapInstance`) could be cached, so they are not called over and over again, but for the time being it is not worth doing. It is on the slow code path and chances are I break something other.

@wwmayer, I put this change as a separate commit in #12577 (one commit per change works better for me) so these two are almost identical. Decision yours, of course.